tensorflow/c/eager/BUILD

```diff
@@ -321,10 +322,31 @@ filegroup(
     srcs = [
         "c_api.h",
         "c_api_experimental.h",
+        "dlpack.h",
     ],
     visibility = ["//tensorflow:__subpackages__"],
 )
 
+
+cc_library(
+    name = "dlpack",
+    srcs = ["dlpack.cc"],
+    hdrs = ["dlpack.h"],
+    copts = [
+        "-fexceptions",
+        "-fno-strict-aliasing",
+    ],
+    features = ["-use_header_modules"],
+    visibility = ["//tensorflow:__subpackages__"],
+    deps = [
+        ":c_api",
+        ":c_api_experimental",
+        "//tensorflow/core:framework",
+        "@dlpack",
```

Comment:
Note: the android CI is failing with

```
ERROR: /tmpfs/src/github/tensorflow/tensorflow/core/BUILD:1376:1: C++ compilation of rule '//tensorflow/core:portable_tensorflow_lib_lite' failed (Exit 1) clang failed: error executing command external/androidndk/ndk/toolchains/llvm/prebuilt/linux-x86_64/bin/clang -gcc-toolchain external/androidndk/ndk/toolchains/aarch64-linux-android-4.9/prebuilt/linux-x86_64 -target ... (remaining 103 argument(s) skipped)

Use --sandbox_debug to see verbose messages from the sandbox
tensorflow/c/eager/dlpack.cc:17:10: fatal error: 'include/dlpack/dlpack.h' file not found
#include "include/dlpack/dlpack.h"  // TF:dlpack
         ^~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
```

Reply:
I excluded it, but not sure whether this works. https://github.com/tensorflow/tensorflow/blob/cae654ea9989ddeafa6a02fa9add9d6cf806f848/tensorflow/c/eager/BUILD#L360-L364